lib/lorekeeper/fast_logger.rb

```diff
@@ -56,7 +56,7 @@ def initialize(file)
     # This is part of the standard Logger API, we need this to be compatible
     def add(severity, message_param = nil, progname = nil, &block)
       return true if severity < @level
-      message = message_param || (block && block.call) || progname
+      message = (block && block.call) || message_param || progname
```

Comment:
why the switch here?

Reply:
it seems some handsome guy suggested it

<img width="943" alt="Screen Shot 2022-07-27 at 5 29 13 PM" src="https://user-images.githubusercontent.com/16286071/181376155-c5a02c71-c697-4714-9452-b3230e18c2d8.png">

Reply:
Very handsome indeed